### PR TITLE
Mongo

### PR DESCRIPTION
**Pull request by nikswam**
Dockerized mongo instance and connected

### CODE REVIEW

File: backend/app/__init__.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import os
```
Comment (ramsgoli) [inline backend/app/__init__.py]: why?

File: backend/app/__init__.py [hunk 2]
```diff
@@ -9,15 +10,9 @@ def create_app():
...
 
-
-    """ MOVE DB CONNECTION TO DOCKER INSTANCE (TODO) """
-    connect(db = 'task-scheduler',
-    username = 'admin',
-    password = 'adminpassword',
-    host =  'mongodb://admin:adminpassword@task-scheduler-shard-00-00-x34zf.azure.mongodb.net:27017,task-scheduler-shard-00-01-x34zf.azure.mongodb.net:27017,task-scheduler-shard-00-02-x34zf.azure.mongodb.net:27017/test?ssl=true&replicaSet=task-scheduler-shard-0&authSource=admin&retryWrites=true')    
-    
     
+    """ MOVE DB CONNECTION TO DOCKER INSTANCE (TODO) """
```
Comment (ramsgoli) [inline backend/app/__init__.py]: ?

File: docker-compose.yml [hunk 3]
```diff
@@ -28,3 +28,8 @@ services:
       - "5000:5000"
     volumes: 
       - ./backend:/var/backend
+  mongo:
```
Comment (ramsgoli) [inline docker-compose.yml]: to be consistent with the postgres service, i would call this service `mongo-db`

**Comment by nikswam** [comment]
Addressed comments

**Pull request merged**